lib/memo_wise.rb

```diff
@@ -120,6 +120,14 @@ def memo_wise(method_name_or_hash) # rubocop:disable Metrics/PerceivedComplexity
               MemoWise::InternalAPI.original_class_from_singleton(klass)
             )
           end
+
+          # Ensures a module extended by another class/module still work
+          # e.g. rails `ClassMethods` module
+          if klass.is_a?(Module) && !klass.is_a?(Class)
+            def klass.extended(base)
```

Comment:
Do you mind adding a comment around why we're using `extended` here vs `included` or `prepended`?